clusters/prodpublick8s.yaml

```diff
@@ -253,3 +253,9 @@ releases:
       - "../config/rating.yaml"
     secrets:
       - "../secrets/config/rating/secrets.yaml"
+  - name: jenkinsisthewayio-redirect
+    namespace: jenkinsisthewayio
```

Comment:
```suggestion
    namespace: jenkinsisthewayio-redirector
```

Reply:
the suffix was made to make it explicit that it does not host any application, only a redirection

Reply:
Which suffix? `io`?

Reply:
> Which suffix? io?

Oh sorry my brain inverted up the 2 lines. Nevermind my comment.

Reply:
I would even suggest "http-redirections" as namespace name, to regroup all redirections.